docs/source/app-dev/custom-views/index.rst

```diff
@@ -326,3 +326,39 @@ A default dedicated dispatcher for blocking operations (e.g. db operation) is co
         throughput = 1
       }
     }
+
+Ledger API Authorization
+========================
+
+The client must provide an access token when authorization is required by the Ledger.
+For details of ledger authorization, please refer to `Ledger Authorization documentation <https://docs.daml.com/app-dev/authorization.html>`_.
+
+Provide access token to custom-view library
+-------------------------------------------
+
+Applications can provide an access token when setting up the client.
+
+.. code-block:: scala
+
+    val clientSettings = GrpcClientSettings
+      .connectToServiceAt(host, port.value)
+      .withCallCredentials(new LedgerCallCredentials(accessToken))
+    val source = BatchSource.events(clientSettings)
+    val control = Projection.project(source, exercisedEvents)(f)
+
+Provide a newly retrieved access token when the existing one expired
+--------------------------------------------------------------------
```

Comment:
```suggestion
------------------------------------------------------------------------
```

Reply:
ah right. it is too short. added more dashes